### ISSUE TITLE
Browser appears briefly black after switching to it, on realme 7i Android 11 device

### ISSUE THREAD

**Issue by mozzaru** [fork-maintainers/iceraven-browser#552]
### Steps to reproduce

why is it like this on me, like in the realme 7i Android 11 device video

### Expected behaviour

why is it like this on me, like in the realme 7i Android 11 device video

### Actual behaviour

why is it like this on me, like in the realme 7i Android 11 device video

### Device name

realme 7i

### Android version

android 11

### Firefox release type

Firefox Nightly

### Firefox version

iceraven 1.19.0

### Device logs

https://user-images.githubusercontent.com/100791931/212524498-bcf2598a-0446-43b4-8091-7fe593ed4e01.mp4

### Additional information

why is it like this on me, like in the realme 7i Android 11 device video

**Comment by interfect** [comment]
Does it do this *every* time you open the browser from the icon? Maybe it is having to restore the app from some kind of saved state?

Does Mozilla Firefox also exhibit this behavior? Or does it have something to do with Iceraven-specific changes?

**Comment by mozzaru** [comment]
@interfect 
yes, like in the video and Firefox in the play store is also the same, I don't know what to do

**Comment by akliuxingyuan** [comment]
@mozzaru I suggest raise a bug to https://bugzilla.mozilla.org/ if Firefox android have same issue.
